website/preprints/tasks.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import urlparse
+
+import requests
+
+from framework.celery_tasks import app as celery_app
+
+from website import settings
+
+
+@celery_app.task(ignore_results=True)
+def on_preprint_updated(preprint_id):
+    # WARNING: Only perform Read-Only operations in an asynchronous task, until Repeatable Read/Serializable
+    # transactions are implemented in View and Task application layers.
+    from website.models import PreprintService
+    preprint = PreprintService.load(preprint_id)
+
+    if settings.SHARE_URL and settings.SHARE_API_TOKEN:
+        requests.post('{}api/normalizeddata/'.format(settings.SHARE_URL), json={
+            'created_at': datetime.datetime.utcnow().isoformat(),
+            'normalized_data': {
+                '@graph': format_preprint(preprint)
+            },
+        }, headers={'Authorization': 'Bearer {}'.format(settings.SHARE_API_TOKEN)}).raise_for_status()
```

Comment:
Added in https://github.com/CenterForOpenScience/osf.io/pull/6346/commits/69bec39d6be765f48320b57d08c41de7e68844b1